node.go

```diff
@@ -16,40 +17,51 @@ import (
 	"github.com/cosmos/iavl/internal/encoding"
 )
 
+// NodeKey represents a key of node in the DB.
+type NodeKey struct {
+	version int64
```

Comment:
shouldn't version be int32 and nonce be int64 since there're more nonce than version